recipes/default.rb

```diff
@@ -18,23 +18,37 @@
 #
 include_recipe 'chef-sugar'
 
-package 'apt-transport-https' if ubuntu?
+# setup docker apt repo
+include_recipe 'chef-apt-docker'
 
-apt_repository 'docker' do
-  uri 'https://apt.dockerproject.org/repo'
-  trusted true
-  components ["ubuntu-#{node['lsb']['codename']}", 'main']
-  only_if { ubuntu? }
+# IAM role support for tasks
+include_recipe 'sysctl::default'
+include_recipe 'iptables::default'
+sysctl_param 'net.ipv4.conf.all.route_localnet' do
+  value 1
+end
+iptables_rule 'dnat_80_to_51679' do
```

Comment:
Two solutions I can think of but neither is clean.

1) Hardcode the rules docker would create into this recipe.
2) Force a docker restart after an iptables restart causing docker to add its rules ontop of the existing rules.

I don't believe I ran into this issue (could be wrong) b/c we were forcing a reboot after install and 2) was effectively happening.